components/actionbutton/stories/template.js

```diff
@@ -128,3 +128,80 @@ export const Template = ({
 		</button>
 	`;
 };
+
+export const ActionButtonsWithIconOptions = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+    ${Template({
+      ...args,
+			iconName: undefined,
+    }, context )}
+    ${Template({
+      ...args,
+    }, context )}
+    ${Template({
+      ...args,
+      hideLabel: true,
+    }, context )}
+	${Template({
+      ...args,
+      hideLabel: true,
+      hasPopup: "true",
+    }, context )}
+	${Template({
+      ...args,
+			iconName: undefined,
+      hasPopup: "true",
+    }, context )}
+  `,
+}, context );
+
+export const IconOnlyOption = ({
+	...args
+}, context ) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		columnGap: "12px",
+	},
+	content: html`
+	${Template({
+      ...args,
+      hideLabel: true,
+      hasPopup: "true",
+    }, context )}
+	${Template({
+      ...args,
+      hideLabel: true,
+			isQuiet: true,
+      hasPopup: "true",
+    }, context )}`
+});
+
+export const TreatmentTemplate = (args, context) => Container({
+	withBorder: false,
+	direction: "row",
+	wrapperStyles: {
+		rowGap: "12px",
+	},
+	content: html`${[
+		{ isSelected: false, isDisabled: false, heading: "Default"}, 
+		{ isSelected: true, isDisabled: false, heading: "Selected" },
+		{ isSelected: false, isDisabled: true, heading: "Disabled" },
+		{ isSelected: true, isDisabled: true, heading: "Selected + disabled" }
+	].map(({isSelected, isDisabled, heading }) => Container({ 
```

Comment:
Could we add the extra space in a few of these objects, just to be the same on both sides? 😆 

`{ isSelected: false, isDisabled: false, heading: "Default" },`
`.map(({ isSelected, isDisabled, heading })`